hs_core/templates/pages/my-resources.html

```diff
@@ -241,7 +241,7 @@ <h4 class="panel-title">
                             </td>
                             {# Title #}
                             <td>
-                                <strong><a href="{{ res.get_absolute_url }}">{{ res.metadata.title }}</a>
+                                <strong><a href="{{ res.get_absolute_url }}">{{ res.metadata.title.value }}</a>
```

Comment:
The reason this change is needed I think due to this: https://github.com/hydroshare/hydroshare/blob/python3-upgrade/hs_core/models.py#L671

Reply:
I actually found a workaround for that so it's no longer needed.  Basically wrapped the unicode mehtod with a tostring type method.  You can see it in a later commit.